cli/cmd/install.go

```diff
@@ -50,6 +50,7 @@ var (
 		"templates/policy/server.yaml",
 		"templates/policy/server-authorization.yaml",
 		"templates/serviceprofile.yaml",
+		"templates/httproute.yaml",
```

Comment:
Why don't we stash this in a directory so it's clearer that it's not part of linkerd?

```suggestion
		"templates/gateway.networking.k8s.io/httproute.yaml",
```

Reply:
also, let's sort this list (so this should be first)